handler/src/main/java/io/netty/handler/ssl/OpenSslCertificateException.java

```diff
@@ -70,10 +424,16 @@ public int errorCode() {
     }
 
     private static int checkErrorCode(int errorCode) {
-        if (errorCode < CertificateVerifier.X509_V_OK || errorCode > CertificateVerifier.X509_V_ERR_DANE_NO_MATCH) {
-            throw new IllegalArgumentException("errorCode must be " + CertificateVerifier.X509_V_OK + " => "
-                    + CertificateVerifier.X509_V_ERR_DANE_NO_MATCH +
-                    ". See https://www.openssl.org/docs/manmaster/apps/verify.html .");
+        if (ERROR_MAP == null) {
+            // We could not load OpenSSL before, so fail here.
+            OpenSsl.ensureAvailability();
+
+            // We should never reach here.
+            assert false;
+        }
+        if (ERROR_MAP.get(errorCode) != Boolean.TRUE) {
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) NullPointerException might be thrown as 'ERROR_MAP' is nullable here [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS2259)